benches/speed_tests.rs

```diff
@@ -47,17 +43,18 @@ use zipf::ZipfDistribution;
 /// This benchmark compares the performance of recursive and iterative FFT.
 pub fn fft(c: &mut Criterion) {
```

Comment:
FWIW, I don't think the FFT benchmarks are all that useful. The goal here was to compare the speed of the iterative FFT algorithm I wrote to the recursive algorithm from the original llbiprio repo.